scalastyle-config.xml

```diff
@@ -60,8 +61,8 @@ This file is divided into 3 sections:
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
- * limitations under the License.
- */]]></parameter>
+ * limitations under the License.\E[\w\W]*\Q
+ */\E]]></parameter>
```

Comment:
Most of the header is still mandatory, but a note can now be added at the end. I needed it to add a modification note to some files (Apache 2.0 licence requirement).